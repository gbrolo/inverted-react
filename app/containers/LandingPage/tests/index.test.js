/**
 *
 * Tests for LandingPage 
 *
 */

import React from 'react';
import { LandingPage } from '../index';
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

describe('<LandingPage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<LandingPage />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
