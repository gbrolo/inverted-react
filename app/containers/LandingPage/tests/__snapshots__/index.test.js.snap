// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> should render and match the snapshot 1`] = `
<div
  className="wrapper flex-centered"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      String Reverser
    </title>
    <meta
      content="Reverse your strings here!"
      name="description"
    />
  </HelmetWrapper>
</div>
`;
